hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/index/bloom/TestHoodieBloomIndex.java

```diff
@@ -428,7 +455,99 @@ public void testTagLocation(boolean rangePruning, boolean treeFiltering, boolean
 
   @ParameterizedTest(name = TEST_NAME_WITH_PARAMS)
   @MethodSource("configParams")
-  public void testCheckExists(boolean rangePruning, boolean treeFiltering, boolean bucketizedChecking) throws Exception {
+  public void testTagLocationOnNonpartitionedTable(
+      boolean rangePruning, boolean treeFiltering, boolean bucketizedChecking,
+      boolean useMetadataTable) throws Exception {
+    // We have some records to be tagged (two different partitions)
+    String rowKey1 = UUID.randomUUID().toString();
```

Comment:
Let's generate UUID from a random w/ fixed seed (so that they don't change from run to run), there's `genPseudoRandomUUID` specifically for that

Reply:
+1
Would be good to do this for all tests.
Just take the change in `FileSystemTestUtils` https://github.com/apache/hudi/pull/6049/files#diff-39cc6a706b7cad836c15caf1a61256c5f50d4ac8ea511de5a4a29f8deb3a2f6c
```
public static final Random RANDOM = new Random(0xDEED);
```

Reply:
Sg.  I make `genPseudoRandomUUID()` public and use that with a pseudo random to generate all UUIDs in this test class.